lib/Model/IMessage.php

```diff
@@ -130,6 +130,12 @@ public function getCloudAttachments(): array;
 	 */
 	public function getLocalAttachments(): array;
 
+	/**
+	 * @param string $name
+	 * @param string $content
+	 */
+	public function addAttachmentFromAttachment(string $name, string $content);
```

Comment:
```suggestion
	public function forwardAttachment(string $name, string $content);
```

:speak_no_evil:

Reply:
and return type can be specified as `void`

Reply:
`forwardAttachment` is better, but wouldn't ´addForwardedAttachment´ betterer?

Reply:
yes :)

Reply:
it's definitely the betterestest option

Reply:
🤣